app/validators/user.js

```diff
@@ -0,0 +1,19 @@
+const { check } = require('express-validator');
+const { defineValidator } = require('./utils');
+const { users: User } = require('../models');
+
+exports.userValidator = defineValidator([
+  check('password')
+    .isLength({ min: 8 })
+    .custom(value => value.match(/^[A-Za-z0-9]*$/)),
+  check('email', 'Email does not is from Wolox').custom(value =>
+    value.match(/([\w_-]*@wolox.[a-z]{3}(.[a-z]*)?)/)
+  ),
+  check('email').custom(email =>
+    User.findOne({ where: { email } }).then(user => {
+      if (user) {
+        throw new Error('E-mail already in use');
+      }
+    })
+  )
+]);
```

Comment:
The validator seems fine to me, but keep in mind that we usually store this kind of pre-controller validations in a `/middlewares` folder, not int `/validators`